facebook/OpenBIC
# common: shell: Add shell power commands

Summary:
- Add shell power commands
  - Get power status
    --> platform power status
  - Control power
    --> platform power control [device_name] [status]
    * **[device_name]**: host/bmc
    * **[status]**: on/off/reset

Test Plan:
- Build Code: PASS
- Check command results in BIC console: PASS

Log:
- BIC console:
  ```
  uart:~$ platform power status
  ------------------------------------
  [HOST]
  * status:      on
  * cpu status:  on
  * post status: on
  [BMC]
  * status: non-support
  ------------------------------------

  uart:~$ platform power control host off
  HOST power off successed!

  uart:~$ platform power status
  ------------------------------------
  [HOST]
  * status:      off
  * cpu status:  off
  * post status: off
  [BMC]
  * status: non-support
  ------------------------------------
  ```
